acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/signing/ValueTransferAcceptanceTest.java

```diff
@@ -35,10 +36,10 @@
 public class ValueTransferAcceptanceTest extends AcceptanceTestBase {
 
   private static final String RECIPIENT = "0x1b00ba00ca00bb00aa00bc00be00ac00ca00da00";
-  private static final long FIFTY_TRANSACTIONS = 50;
+  private static final long NO_OF_TRANSACTIONS = 50;
 
   @Test
-  public void valueTransfer() {
+  public void valueTransfer() throws IOException {
```

Comment:
Don't need the throws IOException on these tests

Reply:
Done